cmd/mount.go

```diff
@@ -313,6 +314,10 @@ var mountCmd = &cobra.Command{
 			options.Components = pipeline
 		}
 
+		if config.IsSet("entry_cache.timeout-sec") || options.EntryCache > 0 {
+			options.Components = append(options.Components[:1], append([]string{"entry_cache"}, options.Components[1:]...)...)
+		}
```

Comment:
Why are we doing this explicitly here? config.Unmarshal should do this

Reply:
Is this for the case when entry cache timeout is passed as CLI parameter?

Reply:
If user pass the entry-cache timeout on cli but did not mention the component in the pipeline I am injecting the compoent explicitly in the pipeline to reduce the user configuration. Ideally we should have done this for other components as well.